DQM/L1TMonitor/src/L1TObjectsTiming.cc

```diff
@@ -0,0 +1,480 @@
+#include "DQM/L1TMonitor/interface/L1TObjectsTiming.h"
+
+
+L1TObjectsTiming::L1TObjectsTiming(const edm::ParameterSet& ps)
+    : ugmtMuonToken(consumes<l1t::MuonBxCollection>(ps.getParameter<edm::InputTag>("muonProducer"))),
+      stage2CaloLayer2JetToken_(consumes<l1t::JetBxCollection>(ps.getParameter<edm::InputTag>("stage2CaloLayer2JetProducer"))),
+      stage2CaloLayer2EGammaToken_(consumes<l1t::EGammaBxCollection>(ps.getParameter<edm::InputTag>("stage2CaloLayer2EGammaProducer"))),
+      stage2CaloLayer2TauToken_(consumes<l1t::TauBxCollection>(ps.getParameter<edm::InputTag>("stage2CaloLayer2TauProducer"))),
+      stage2CaloLayer2EtSumToken_(consumes<l1t::EtSumBxCollection>(ps.getParameter<edm::InputTag>("stage2CaloLayer2EtSumProducer"))),
+      l1tStage2uGtProducer_(consumes<GlobalAlgBlkBxCollection>(ps.getParameter<edm::InputTag>("ugtProducer"))),
+      monitorDir(ps.getUntrackedParameter<std::string>("monitorDir")),
+      verbose(ps.getUntrackedParameter<bool>("verbose")),
```

Comment:
Construct the gtUtils_ here and erase the first five lines in the constructor implementation.
`gtUtil_(new l1t::L1TGlobalUtil(params, consumesCollector(), *this, params.getParameter<edm::InputTag>("l1tStage2uGtSource"), params.getParameter<edm::InputTag>("l1tStage2uGtSource"))),`